src/rubrix/server/apis/v0/models/text2text.py

```diff
@@ -215,6 +215,7 @@ class Text2TextSearchRequest(BaseModel):
 
     query: Text2TextQuery = Field(default_factory=Text2TextQuery)
     sort: List[SortableField] = Field(default_factory=list)
+    id_from: Optional[str] = None
```

Comment:
I think we can drop this field, since we're passing the `id_from` as id param instead of a body attibute